buildkite/src/Command/DeployTestnet.dhall

```diff
@@ -0,0 +1,42 @@
+let Prelude = ../External/Prelude.dhall
+
+let Command = ./Base.dhall
+let Size = ./Size.dhall
+
+let Cmd = ../Lib/Cmds.dhall
+
+let defaultArtifactStep = { name = "MinaArtifact", key = "mina-docker-image" }
+
+let deployEnv = "DOCKER_DEPLOY_ENV" in
+
+{ step = \(testnetName : Text) -> \(dependsOn : List Command.TaggedKey.Type) -> \(postDeploy : Text) ->
+    Command.build
+      Command.Config::{
+        commands = [
+          Cmd.run (
```

Comment:
not urgent, but we could consider moving this to the scripts folder

Reply:
No doubt and I can see either way.

The thinking here was to avoid having to create a script for jobs/steps when necessary. Buildkite command steps inherently create an execution environment for steps so seems a bit redundant to create another embedded within this with a script. Also avoids code clutter with added scripts and the # of code points to reference.